osbs/schemas/container.json

```diff
@@ -335,6 +335,11 @@
             "required": ["registry", "package_mappings"],
             "additionalProperties": false
           }
+        },
+        "skip_all": {
+          "description": "Skips all modifications of CSV operator file, allowed only for defined koji packages",
```

Comment:
`defined` where? It should be explained in description

Reply:
updated